.github/workflows/examples.yaml

```diff
@@ -0,0 +1,82 @@
+name: Run examples on push
+
+on:
+  push:
+    branches:
+    - master
+    - develop
+  pull_request:
+    branches:
+    - master
+    - develop
+
+jobs:
+  run-examples:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14.15.4'
+      - name: Cache Node.js modules
+        uses: actions/cache@v2
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install python dependencies
+        run: python -m pip install pipenv
+      - run: pipenv sync
+      - name: Install dependencies
+        run: pipenv run yarn install
+      - name: Execute lint
+        run: yarn lint
```

Comment:
Could it be possible we're seeing multiple lint warnings as we're executing lint in multiple workflows?

Reply:
maybe, do we need a lint in examples ? i think we can remove this